### PR TITLE
fix: sort by tx and filter out ens + uniswap mints

### PR DESCRIPTION
**Pull request by lunarmayor**

### CODE REVIEW

File: packages/indexer/src/elasticsearch/indexes/activities/index.ts [hunk 1]
```diff
@@ -182,6 +182,12 @@ export enum TopSellingFillOptions {
   any = "any",
 }
 
+const trendingBlocklist = [
```
Comment (nofir) [inline packages/indexer/src/elasticsearch/indexes/activities/index.ts:185]: if its only relevant for eth mainnet, maybe have it in the network settings for the index:
https://github.com/reservoirprotocol/indexer/blob/8b624d6a6a4c33e6c850e4e3c43f13e795607d49/packages/indexer/src/config/network.ts#L199

**Review by nofir — CHANGES_REQUESTED** [review]

File: packages/indexer/src/elasticsearch/indexes/activities/index.ts (hunk 1 above)
Comment (lunarmayor) [reply packages/indexer/src/elasticsearch/indexes/activities/index.ts]: sorry took a bit but made the change

Comment (lunarmayor) [reply packages/indexer/src/elasticsearch/indexes/activities/index.ts]: @nofir good to merge?

**Review by nofir — APPROVED** [review]

File: packages/indexer/src/elasticsearch/indexes/activities/index.ts (hunk 1 above)
Comment (nofir) [reply packages/indexer/src/elasticsearch/indexes/activities/index.ts]: approved

**Pull request merged**